### PR TITLE
Change SedAAdjust factor

### PR DESCRIPTION
**Pull request by kellyi**
## Overview

This  PR changes the SedAAdjust factor from 1 -> 1.4 as described in #1974. It also adds some text to the MapShed model tabs describing the adjustment

Connects #1974 

### Demo

screen shot 2017-07-10 at 5 36 53 pm

## Testing Instructions

* get this branch then `bundle`
* run mapshed on an aoi and verify that it still works and that you see the new text added beneath the hydrology and water quality tabs

### CODE REVIEW

**Comment by kellyi** [comment]
Tagging @ajrobbins in case we need to move this text or also add it elsewhere.

**Comment by ajrobbins** [comment]
@jfrankl Where would be the best place for this text?

**Comment by jfrankl** [comment]
I think a good place for this would be a tooltip on the text at the top of the sidebar. It makes sense to pair this with the other text describing the tab, but it's too much text to include without containing it in a tooltip or a similar method.

image

**Comment by ajrobbins** [comment]
Yep, looks great!

**Comment by kellyi** [comment]
Implemented these tooltips for the runoff and water quality pages:

screen shot 2017-07-12 at 3 39 20 pm

**Comment by kellyi** [comment]
This one's ready for review!

File: src/mmw/js/src/modeling/gwlfe/runoff/views.js [hunk 1]
```diff
@@ -75,6 +77,10 @@ var ResultView = Marionette.LayoutView.extend({
         }
     },
 
+    onRender() {
```
Comment (rajadain) [inline src/mmw/js/src/modeling/gwlfe/runoff/views.js]: This isn't ES6. Use

```js
onRender: function() {
```

instead.

File: src/mmw/js/src/modeling/gwlfe/quality/views.js [hunk 2]
```diff
@@ -37,6 +42,17 @@ var ResultView = Marionette.LayoutView.extend({
                 }));
             }
         }
+    },
+
+    onRender() {
```
Comment (rajadain) [inline src/mmw/js/src/modeling/gwlfe/quality/views.js]: Here too.

File: src/mmw/js/src/modeling/gwlfe/runoff/views.js (hunk 1 above)
Comment (kellyi) [inline src/mmw/js/src/modeling/gwlfe/runoff/views.js]: Good catch -- I meant to replace these before pushing so I'll make a fixup commit for that.

Comment (rajadain) [inline src/mmw/js/src/modeling/gwlfe/runoff/views.js]: You can run `./scripts/npm.sh run lint` locally to see warnings for this and other things that can make a PR "Unstable" in Jenkins

File: src/mmw/sass/pages/_model.scss [hunk 3]
```diff
@@ -61,4 +61,8 @@
       }
     }
   }
+
+  .model-results-tooltip {
+    color: #333333;
```
Comment (arottersman) [inline src/mmw/sass/pages/_model.scss]: The icon get's a little cutoff
screen shot 2017-07-13 at 8 24 14 am

Consider adding a width to it
```css
i {
    width: 15px;
}
```

**Pull request merged**